library/net.c

```diff
@@ -203,7 +207,11 @@ int mbedtls_net_bind( mbedtls_net_context *ctx, const char *bind_ip, const char
 
         n = 1;
         if( setsockopt( ctx->fd, SOL_SOCKET, SO_REUSEADDR,
+#if VXWORKS
+                        (char *) &n, sizeof( n ) ) != 0 )
+#else
```

Comment:
Unfortunately not true in practice with the current shipping VxWorks 6.7 compiler.

I know, pretty ugly. The only other solution would be to introduce a typedef for the setsockopt arg. Equally ugly.